bin/katello-certs-check

```diff
@@ -178,6 +178,20 @@ function check-cert-san () {
     CHECK=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)
     if [[ $? == "0" ]]; then
         success
+        printf "Checking if any of Subject Alt Names on certificate matches the Subject CN"
```

Comment:
```suggestion
        printf "Checking if any Subject Alt Name on certificate matches the Subject CN"
```

Just a nitpick

Reply:
That's fine. I was not 100% comfortable with my initial wording.